07-DialogService/src/PrismSample.Android/Resources/values/styles.xml

```diff
@@ -11,7 +11,7 @@
     <item name="windowActionBar">false</item>
     <!-- Set theme colors from https://aka.ms/material-colors -->
     <!-- colorPrimary is used for the default action bar background -->
-    <item name="colorPrimary">#2196F3</item>
+    <item name="colorPrimary">#00A5F2</item>
```

Comment:
no style changes please

Reply:
Damn it. It should be reverted properly now 👍